apps/emqx/src/emqx_cm.erl

```diff
@@ -189,7 +189,7 @@ do_unregister_channel({_ClientId, ChanPid} = Chan) ->
     true = ets:delete(?CHAN_CONN_TAB, Chan),
     true = ets:delete(?CHAN_INFO_TAB, Chan),
     ets:delete_object(?CHAN_TAB, Chan),
-    ok = emqx_hooks:run('channel.unregistered', [ChanPid]),
+    ok = emqx_hooks:run('cm.channel.unregistered', [ChanPid]),
```

Comment:
Nit. Why did this need changing?

Reply:
I tried to emphasize that this hook is **not** evaluated in the context of a client/channel